### DISCUSSION TITLE
Why "Immich"?

### DISCUSSION THREAD

**Discussion by Chiogros** [immich-app/immich#158] [Q&A] [answered]
Hi, what does "Immich" mean?
Thanks :)

**Comment by alextran1502** [comment, marked as answer]
It is pronounced as "image" :)

**Comment by Cantello** [comment]
Maybe, once it's production-ready (or better slightly before that) it could be wise to change the name to one that better rolls off the tongue. It's always a bad sign if one has to ask how something is pronounced... :-)

But even so, I will continue using Immich, thanks for this great piece of work!

**Reply by alextran1502** [reply]
I don't think pronouncing "image" is hard 😅

Plus I don't mind to stay unique. Brand like Peugeot, Bvlgari is also dazzling to first time people who know about them on how to pronounce the name 😜

**Reply by Cantello** [reply]
You are of course right, I still have to think about how to pronounce it and in my native tongue it reads a bit weird. 😜

**Reply by alextran1502** [reply]
Just curious, how does it sound in your native tongue, and what language do you speak natively? 

In my native tongue (Vietnamese) it sounds exactly the same as how to pronounce Immich in English so I didn't think about this very hard 😅

**Reply by xpwmaosldk** [reply]
In my native language(Korean), 
`image` is pronounced `imiji` 
`immich` is pronounced `Im Michi`😜

**Reply by Cantello** [reply]
In German I always read it as 'im ich' (=in the me/ego). I would pronounce it like ['ɪm 'ɪ​ç].

**Comment by xkx** [comment]
I read it as 'Imrich'.

**Reply by Nonobis** [reply]
immich in french, double 'm' not pronounced

**Comment by noapoleon** [comment]
not sure why but i assumed it was pronounced like in german: https://youtu.be/aVXi2PrE8u0?si=lRNxTtokE6w43Oth
